src/single-spa-angular2.js

```diff
@@ -45,18 +41,29 @@ function bootstrap(opts) {
   return Promise.resolve();
 }
 
-function mount(opts) {
-  const containerEl = getContainerEl(opts);
-  containerEl.innerHTML = opts.template;
-  return opts
-    .angularPlatform
-    .bootstrapModule(opts.mainModule)
-    .then(module => {
-      return opts.bootstrappedModule = module;
+function mount(opts, props) {
+  return Promise
+    .resolve()
+    .then(() => {
+      const domElementGetter = chooseDomElementGetter(opts, props)
+      if (!domElementGetter) {
+        throw new Error(`cannot mount angular application '${props.name || props.appName}' without a domElementGetter provided either as an opt or a prop`)
+      }
+
+      const containerEl = getContainerEl(domElementGetter)
+      containerEl.innerHTML = opts.template
```

Comment:
do you need to be returning something in this promise for the next `.then(domElementGetter => ...` to work?

hm, maybe not, since you're not actually using domElementGetter in the next .then. so maybe just remove it so you're good?

Reply:
yeah I thought i was going to use domElementGetter but I'm not. Will change 👍